### DISCUSSION TITLE
View the findings from our community survey (Oct 2024)

### DISCUSSION THREAD

**Discussion by jricciardi** [community/community#144972] [Announcements]
We love hearing from our members and we love sharing back what we learned from you from our surveys. Below is an infographic round-up of the key findings and most common feedback from the October 2024 Community survey!

Thank you again to every community member who participated! We do look at every single response and use this to inform some of our strategy going forward to make the GitHub Community Discussions the best they can be.

![community-infographic2024oct](https://github.com/user-attachments/assets/db912048-e362-444a-9ac2-f076744c8b92)

# GitHub Community Sentiment Survey - November 2024 Key Findings Alt Text

## Headline Numbers
- Total Community solutions: 9,212
- Median time to solution: 9 hours
- 80% of users say the Community "Meets or exceeds expectations"

## Community Priorities
1. Skill-building: 25%
2. Networking: 22%
3. Industry Discussion: 20%
4. AMAs: 18%
5. Non-tech Discussions: 15%

## Most Valued Community Aspects
- Quick and respectful responses from users
- Helpful community forums for bug fixing
- Platform understanding and improvement
- Developer skill development opportunities

## Areas of Opportunity
- Need for more non-English language moderators
- Search functionality enhancements requested
- Better filtering options and keyword suggestions
- 25% of users specifically want more skill-building opportunities

Thank you!

We regularly run this survey to you so we can continuously improve your community experience. Be on the lookout for the next one in about 6 months 👋

**Comment by Samunng** [comment]
قناة القدس اليوم الفضائية
ميرات السماء
                                                       www.alqudstoday.tv

**Comment by Samunng** [comment]
موقع قناة السلام عليك ايها النبي الفضائية
                                               www.assalamu-alayka.tv

**Comment by zaphito** [comment]
[]()

**Comment by Amir429206** [comment]
I
